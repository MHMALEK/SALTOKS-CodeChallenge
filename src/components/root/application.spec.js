import React from "react"
import renderer from "react-test-renderer"
import { MemoryRouter } from "react-router-dom"
import { Application } from "./Application"
import mockRedux from "../../../__mocks__/redux"
describe("SnappShot for Application", () => {
  const mockProps = {
    bootsrapApp: () => mockRedux()
  }
  test("snapshot renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <Application {...mockProps} />
      </MemoryRouter>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
