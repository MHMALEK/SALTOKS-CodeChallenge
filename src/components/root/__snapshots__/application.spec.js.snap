// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnappShot for Application snapshot renders 1`] = `
Array [
  " ",
  " ",
]
`;
